### ISSUE TITLE
Charts should be published and indexed on merge to master

### ISSUE THREAD

**Issue by max-rocket-internet** [aws/eks-charts#53]
It's just confusing for everyone otherwise.

For example:
https://github.com/aws/eks-charts/issues/46
https://github.com/aws/eks-charts/issues/36

**Comment by jaypipes** [comment]
Not sure about every merge to master, but I suppose I could support auto-publish/index when a new tag is pushed to the master repos of the underlying projects

**Comment by max-rocket-internet** [comment]
No, I mean like someone makes a PR to update a chart in this repo. The PR is reviewed, the chart version is bumped and the the PR is merged. At this point, we can see it in master but we can't use it as it hasn't been published yet. That is what those 2 issues above are about.

My point is: once a PR has been approved and merged, it should be published and available to use via Helm immediately. That's how the main stable charts repo does it. It's confusing if people have to wait until a tag is created for this whole repo. See what I mean?

**Comment by stefanprodan** [comment]
It takes many hours for the stable charts to be published on GPC. The current publishing flow allows us to bundle several PRs in a release and have a change log in GitHub releases. I think it's the reviewer responsibility to do a release or not after the merge and also document the changes.

**Comment by max-rocket-internet** [comment]
> It takes many hours for the stable charts to be published on GPC

It takes less than a minute in our CI with many more charts, feel free to copy it: https://github.com/deliveryhero/helm-charts/blob/master/.github/workflows/push-chart.yaml

😃

**Comment by max-rocket-internet** [comment]
@jaypipes @M00nF1sh @fawadkhaliq can we address what I mention in this issue?

It's so frustrating having to push to get a chart update merged and THEN have to wait/push for a new release to be made.

Another example is https://github.com/aws/eks-charts/pull/345

> Not sure about every merge to master

Why not? That's a totally normal work flow. The old Helm stable repo [did this](https://github.com/helm/charts/blob/2e46bf07846e18e7ce844ed580e8df8193d47a6d/.github/workflows/sync-repos.yaml#L4) and I don't know how bitnami do it but a [PR merged 2 hours ago](https://github.com/bitnami/charts/pull/4620) is already in their index 😅

**Comment by max-rocket-internet** [comment]
@kishorj @bwagner5

Look at all these issues that could be prevented:

https://github.com/aws/eks-charts/issues/272
https://github.com/aws/eks-charts/issues/281
https://github.com/aws/eks-charts/issues/247
https://github.com/aws/eks-charts/issues/244

**Comment by max-rocket-internet** [comment]
And more

https://github.com/aws/eks-charts/issues/185
https://github.com/aws/eks-charts/issues/193

I think you get my point 🙂

**Comment by jwenz723** [comment]
I would love to see this functionality built in as well. My PR, #472, got merged 11 days ago. I still cannot make use of the features in my PR because a released version of the helm chart has not been published yet.

**Comment by kazu** [comment]
when will the newer version including the aws-for-fluent-bitchart  (> 0.1.7) publish?
these PR already is merged to master

#446
#492
#353
#491
